wolfcrypt/src/poly1305.c

```diff
@@ -18,11 +18,12 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
  */
-
 /*
- * Based off the public domain implementations by Andrew Moon
```

Comment:
Can you please retain the "based off" info?

Reply:
@dgarske, Done